docker-compose.yml

```diff
@@ -40,6 +40,26 @@ services:
       - "8086:8086"
     entrypoint: "gcloud beta emulators bigtable start"
 
+  learning_service:
+    build:
+      context: ./learning_service
+      dockerfile: Dockerfile
+    image: learning_service:0
```

Comment:
Specifying `image` and `container_name` in the Compose file is a bit superfluous, these Compose files are usually used mainly to test out the "mini-production" environment locally and are tagged with respect to the Compose project name (taken from the name of the directory in which the Compose file resides).